framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -482,16 +527,77 @@ private AnnotationMirror getAnnotationMirrorOfMetaAnnotation(
         for (Pair<AnnotationMirror, AnnotationMirror> r : declAnnotations) {
             AnnotationMirror anno = r.first;
             AnnotationMirror metaAnno = r.second;
-            List<String> expressions =
-                    AnnotationUtils.getElementValueArray(anno, "expression", String.class, false);
-            AnnotationMirror postcondAnno = getAnnotationMirrorOfMetaAnnotation(metaAnno, anno);
-            if (postcondAnno == null) {
-                continue;
-            }
-            boolean annoResult =
-                    AnnotationUtils.getElementValue(anno, "result", Boolean.class, false);
-            for (String expr : expressions) {
-                result.add(new ConditionalPostcondition(expr, annoResult, postcondAnno, anno));
+
+            result.addAll(getConditionalPostConditionAnnotations(anno, metaAnno));
+        }
+
+        return result;
+    }
+
+    /**
+     * Returns conditional postcondition annotations.
```

Comment:
Similar comment to above about comment:  initial comment is too vague, `methodElement` doesn't exist.
Make all suggestions from elsewhere to this method (documentation and code).

Reply:
I have updated the documentation. Please review it!